package.json

```diff
@@ -5,6 +5,7 @@
   "main": "src/index.js",
   "files": [
     "src",
+    "src/middleware",
```

Comment:
This works, but it's strange. I wouldn't expect to need to include `src` in the import path.